runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -2257,6 +2257,32 @@ TR_J9ServerVM::delegatingMethodHandleTargetHelper(TR::Compilation *comp, TR::Kno
    knot->updateKnownObjectTableAtServer(idx, std::get<1>(recv));
    return idx;
    }
+
+UDATA
+TR_J9ServerVM::getVMTargetOffset()
+   {
+   JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
+   auto *vmInfo = _compInfoPT->getClientData()->getOrCacheVMInfo(stream);
+   if (vmInfo->_vmtargetOffset)
```

Comment:
is it guaranteed that a value of 0 is not valid?

Reply:
I do not know if it is guaranteed, but I've never seen a valid value be 0.
In the worst-case scenario, if the valid value is 0, we'll just end up sending a message every time this query is called